### ISSUE TITLE
States Language Support

### ISSUE THREAD

**Issue by samgaw** [alyssaxuu/flowy#25]
This may be way outside the scope of the project but it would be awesome to reuse data specs already available like [States Language](https://github.com/awslabs/states-language) and augment the spec with the presentation set used in flowy e.g.
```json
"States": {
  "BlockElem1": {
    "Type": "Trigger",
    "Parameters": {
      "flowy": {
        "x": 850,
        "y": 505,
        "width": 318,
        "height": 120
      }
    }
  },
  "BlockElem2": { ... }
}
```

**Comment by alyssaxuu** [comment]
Could be added in, although would have to rewrite a lot of it, specially for the saving / loading section which is a bit more complicated than just changing the names of the keys and whatnot.

**Comment by samgaw** [comment]
Yeah, that was my main reason framing it as possibly way outside of the project's scope. However as the project becomes more popular, requests like #28 & #29 are likely to become more frequent. There's going to be significant creep as more people project their individual requirements. Not reinventing the wheel, and instead making use of a well reviewed spec, defines the functional boundaries for the library and makes the edge cases a presentation matter.
